docs/CodeStyle.md

```diff
@@ -32,6 +32,14 @@
 ## C++
   * used only for unit testing
   * lower-case class names (same as stl/boost)
+  * the unit tests in test/gtest are written using [C++11](https://en.cppreference.com/w/cpp/11). Whenever applicable, the usage of advanced language features is allowed and preferred over legacy code. For example:
```

Comment:
`language features is allowed and preferred` - I'm not sure about interpretation of this. Do you want just encourage the usage of C++/11 ? Or the plan is to reject patches written in "legacy" (whatever this means for c++)

Reply:
patches that use a legacy style where it's obviously possible and simple to use C++11, will be most likely rejected with the appropriate review comments